apps/editor.planx.uk/src/components/EditorNavMenu/EditorNavMenu.tsx

```diff
@@ -133,7 +133,7 @@ function EditorNavMenu() {
         : `Planning Data unavailable`,
       Icon: LayersIcon,
       route: referenceCode
-        ? `https://submit.planning.data.gov.uk/organisations/local-authority:${referenceCode}`
+        ? `https://provide.planning.data.gov.uk/organisations/local-authority:${referenceCode}`
```

Comment:
Looks like PD has since updated this URL, our previous one was automatically redirecting